rosdoc2/verbs/build/builders/sphinx_builder.py

```diff
@@ -78,23 +78,22 @@ def ensure_global(name, default):
     from exhale import utils
     exhale_args.update({{
         # These arguments are required.
-        "containmentFolder": "{user_sourcedir}/api",
-        "rootFileName": "library_root.rst",
-        "rootFileTitle": "{package_name} API",
+        "containmentFolder": "{user_sourcedir}/generated",
+        "rootFileName": "index.rst",
         "doxygenStripFromPath": "..",
         # Suggested optional arguments.
         "createTreeView": True,
+        "fullToctreeMaxDepth": 1,
+        "unabridgedOrphanKinds": [],
+        "fullApiSubSectionTitle": "Reference",
         # TIP: if using the sphinx-bootstrap-theme, you need
         # "treeViewIsBootstrap": True,
         "exhaleExecutesDoxygen": False,
         # Maps markdown files to the "md" lexer, and not the "markdown" lexer
         # Pygments registers "md" as a valid markdown lexer, and not "markdown"
         "lexerMapping": {{r".*\.(md|markdown)$": "md",}},
-        # This mapping will work when `exhale` supports `:doxygenpage:` directives
-        # Check https://github.com/svenevs/exhale/issues/111
-        # TODO(aprotyas): Uncomment the mapping below once the above issue is resolved.
-        # "customSpecificationsMapping": utils.makeCustomSpecificationsMapping(
-        #     lambda kind: [":project:", ":path:", ":content-only:"] if kind == "page" else []),
+        "customSpecificationsMapping": utils.makeCustomSpecificationsMapping(
+            lambda kind: [":content-only:", ":no-link:"] if kind == "page" else []),
```

Comment:
IIRC, this also changes the behavior when there is a different (not `:doxygenpage:`) directive. As in, other directives such as `:doxygenfunction:` wouldn't render correctly with this mapping, and I think you have to provide mappings for each such directive. I will confirm whether that's true in a future comment.

Reply:
:100: in general for the behavior this should give otherwise though.